selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java

```diff
@@ -1911,10 +1911,10 @@ public void likeReply(String reply) {
     public static void uploadFileFromAS(String fileName) {
         // Make sure the Activity Composer File tab is not hidden by the top bar
         executeJavaScript("window.scrollTo(0,0)");
-        ELEMENT_ACTIVITY_COMPOSER_FILE_TAB.waitUntil(exist, timeout).click();
-        ELEMENT_INPUT_DOCUMENT.waitUntil(exist, timeout).uploadFromClasspath(fileName);
+        ELEMENT_ACTIVITY_COMPOSER_FILE_TAB.waitUntil(exist, openBrowserTimeoutMs).click();
+        ELEMENT_INPUT_DOCUMENT.waitUntil(exist, openBrowserTimeoutMs).uploadFromClasspath(fileName);
         ELEMENT_BAR_PROGRESS.waitUntil(disappears, openBrowserTimeoutMs);
-        $(ELEMENT_COMPOSER_SHARE_BUTTON).waitUntil(enabled, timeout).click();
+        $(ELEMENT_COMPOSER_SHARE_BUTTON).waitUntil(enabled, openBrowserTimeoutMs).click();
```

Comment:
Can you explain this change please ?

Reply:
We should instead understand why it takes so much time. And if it is really necessary to set a higher timeout, we should not use openBrowserTimeoutMs since it is dedicated to open browser use case.